### PR TITLE
Fix/text

### PR DESCRIPTION
**Pull request by Tanguylo**

### CODE REVIEW

File: src/shapes.ts [hunk 1]
```diff
@@ -337,14 +353,11 @@ export class Text extends Shape {
     return this.splitInWords();
   }
 
-  private pushSeparatorInWords(pickedChars: number): [string, number] {
-    return [this.text[pickedChars], pickedChars + 1]
-  }
-
   private buildWord(pickedChars: number, word: string): [string, number] {
```
Comment (GhislainJ) [inline src/shapes.ts:356]: In the img diff, it looks like the linebreaks don't occur at the same place, and sometimes at the middle of a word (separated by a "_" for ex). I guess this comes from this change. Is it the wanted  behavior ?

**Comment by GhislainJ** [comment]
Don't you want it to be a patch ?

File: src/shapes.ts (hunk 1 above)
Comment (Tanguylo) [reply src/shapes.ts:356]: Yes it is wanted because '_' is a separator but I'm open to other opinions :)

**Comment by Tanguylo** [comment]
> Don't you want it to be a patch ?

Don't know yet, it depends on project needs

File: src/shapes.ts (hunk 1 above)
Comment (GhislainJ) [reply src/shapes.ts:356]: No, it's all fine to me. I just wanted to be sure it wasn't a bug introduced

**Comment by Tanguylo** [comment]
@GhislainJ 
The project needs a fix for this bug on the current bundle.
But this PR takes advantage of a Shape refactor, so let's keep it and push it in dev for further updates :)

I am now pushing a new request on chore/support-0-22 branch to hotfix the bug.

**Review by GhislainJ — APPROVED** [review]

**Pull request merged**